core/common/src/main/java/org/phenotips/remote/common/internal/api/DefaultPatientToJSONConverter.java

```diff
@@ -49,6 +49,8 @@
  */
 public class DefaultPatientToJSONConverter implements PatientToJSONConverter
 {
+    private static final String PUBMED_BASE_URL = "https://www.ncbi.nlm.nih.gov/pubmed/?term=";
```

Comment:
This is the Search URL, it is recommended to use the [Retrieve URL](https://www.ncbi.nlm.nih.gov/books/NBK3862/#linkshelp.Retrieve_PubMed_Citations), which means simply removing the `?term=` suffix.